packages/kbn-i18n/package.json

```diff
@@ -2,30 +2,41 @@
   "name": "@kbn/i18n",
   "browser": "./target/web/browser.js",
   "main": "./target/node/index.js",
+  "types": "./target/types/index.d.ts",
   "module": "./src/index.js",
   "version": "1.0.0",
   "license": "Apache-2.0",
   "private": true,
   "scripts": {
-    "build": "yarn build:web && yarn build:node",
-    "build:web": "cross-env BABEL_ENV=web babel src --out-dir target/web",
-    "build:node": "cross-env BABEL_ENV=node babel src --out-dir target/node",
+    "build": "yarn build:web && yarn build:node && yarn build:types",
+    "build:types": "tsc --emitDeclarationOnly",
+    "build:web": "cross-env BABEL_ENV=web babel src --config-file ./babel.config.js --out-dir target/web --extensions \".ts,.js,.tsx\"",
+    "build:node": "cross-env BABEL_ENV=node babel src --config-file ./babel.config.js --out-dir target/node --extensions \".ts,.js,.tsx\"",
     "kbn:bootstrap": "yarn build",
     "kbn:watch": "yarn build --watch"
   },
   "devDependencies": {
-    "@kbn/babel-preset": "link:../kbn-babel-preset",
-    "@kbn/dev-utils": "link:../kbn-dev-utils",
-    "babel-cli": "^6.26.0",
-    "cross-env": "^5.2.0"
+    "@babel/cli": "^7.0.0",
+    "@babel/core": "^7.0.1",
+    "@babel/plugin-proposal-class-properties": "^7.0.0",
+    "@babel/plugin-proposal-object-rest-spread": "^7.0.0",
+    "@babel/preset-env": "^7.0.0",
+    "@babel/preset-react": "^7.0.0",
+    "@babel/preset-typescript": "^7.0.0",
+    "@types/intl-messageformat": "^1.3.0",
```

Comment:
__nit:__ this package is shipped with its own types (https://github.com/yahoo/intl-messageformat/blob/master/index.d.ts), we don't need this "@types" package it seems.